Examples/UIExplorer/NavigationExperimental/NavigationAnimatedExample.js

```diff
@@ -99,8 +100,10 @@ class NavigationAnimatedExample extends React.Component {
   _renderHeader(/*NavigationSceneRendererProps*/ props) {
     return (
       <NavigationHeader
-        {...props}
-        getTitle={state => state.key}
+        navigationProps={props}
```

Comment:
@hedgerwang If I pass it as a separate prop, it's easier to pass it to renderLeftComponent, renderRightComponent etc.

@ericvicenti The object exported from NavigationPropType is still a mix of prop types and plain objects even if I spread